src/components/implementation/cbuf_mgr/default/cbuf_mgr.c

```diff
@@ -295,7 +297,7 @@ cbuf_map(spdid_t spdid, vaddr_t daddr, void *page, unsigned long size, int flags
  * return spdid's address to daddr, manager's virtual address to page
  */
 static int
-cbuf_alloc_map(spdid_t spdid, vaddr_t *daddr, void **page, void *p, unsigned long sz, int flags)
+cbuf_alloc_map(spdid_t spdid, vaddr_t *daddr, void **page, void *p, unsigned long sz, int flags, vaddr_t exact_dest)
```

Comment:
I will consider that. Sounds like a good idea. And seven arguments as written now is usually considered unprofessional...